recipe/meta.yaml

```diff
@@ -41,13 +41,13 @@ requirements:
     - numpy ~=1.17
     - pamqp ~=2.3
     - paramiko ~=2.7,>=2.7.2
-    - plumpy ~=0.19.0
+    - plumpy ~=0.20.0
     - pgsu ~=0.2.0
     - psutil ~=5.6
     - psycopg2-binary ~=2.8.3
```

Comment:
Also need to update this to include `<2.9`

Reply:
Are you sure? I was going by: https://github.com/aiidateam/aiida-core/blob/support/1.6.x/environment.yml

Reply:
nvm, since we `psycopg2-binary ~=2.8.3` that effectively means `<2.9` so its fine.